packages/core/src/App/Containers/SetResidenceModal/set-residence-form.jsx

```diff
@@ -48,14 +49,16 @@ const SetResidenceForm = ({
             </Field>
             {!errors?.residence?.length > 0 ? (
                 <Text as='p' size='xxs' className='account-signup__subtext' color='less-prominent'>
```

Comment:
will this work here?
{(!errors?.residence?.length) && (
    <Text as='p' size='xxs' className='account-signup__subtext' color='less-prominent'>
         <Localize i18n_default_text='Country of residence is where you currently live.' />
    </Text>
)}